server/index.js

```diff
@@ -14,6 +14,12 @@ app.use('/:listingId/', (req, res, next) => {
   }
   next()
 })
+//middleware for CORS headers
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*"); // update to match the domain you will make the request from
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
```

Comment:
NIce! I went about it a different way, installing the cors npm package and then using it as middleware. Cool to see an alternative.